package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.3.2",
   "description": "Material-design \"Snackbar\" component for Android and iOS",
   "main": "src/index.js",
+  "types": "./index.d.ts",
```

Comment:
Could you put this in `src` next to regular `index.js` please? I assume that will work just as well even if the compiled code goes in `lib`.